configure contains CFLAGS filtering code which was removing our prefix-map
flags. We need those to generate reproducible binaries. Allow them through.

Upstream-Status: Pending
Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>

Index: libid3tag-0.15.1b/configure.ac
===================================================================
--- libid3tag-0.15.1b.orig/configure.ac
+++ libid3tag-0.15.1b/configure.ac
@@ -99,6 +99,10 @@ do
 	-mno-cygwin)
 	    shift
 	    ;;
+	-fmacro-prefix-map*|-fdebug-prefix-map*)
+	    CFLAGS="$CFLAGS $1"
+	    shift
+	    ;;
 	-m*)
 	    arch="$arch $1"
 	    shift
